config/core/test-configs.yaml

```diff
@@ -1939,6 +1945,7 @@ test_configs:
       - kselftest-lkdtm
       - kselftest-rtc
       - kselftest-seccomp
+      - kselftest-tpm2
```

Comment:
How about running it on other devices? Does it require particular hardware only available on grunt?

Reply:
Yes, I intend to. I was having a few issues getting the test running on a few other devices, and grunt was the first one I got it working on. But there should be a few other devices in KernelCI where we can run this test on. By the end of next week I should probably be able to get at least one other device running this test.